elastic/kibana
# [Serverless] [Advanced Settings] Remove storeInSessionStorage setting

Closes https://github.com/elastic/kibana/issues/168480

## Summary

This PR removes the `storeInSessionStorage` setting from the serverless common settings allowlist, so that the setting is hidden from Advanced Settings in all serverless projects.

**How to test:**
1. Start Es with `yarn es serverless` and Kibana with `yarn serverless-{es/oblt/security}`
2. Go to Management -> Advanced Settings
3. Verify that the setting is not on the page.
